src/wallets/wallets.service.ts

```diff
@@ -36,12 +36,12 @@ export class WalletsService {
   async uploadKeysToAWS(privateKey: string, publicKey: string, userId: string): Promise<void> {
     const paramsPublic = {
       Bucket: process.env.KEYS_BUCKET_NAME,
-      Key: `${userId}/private.pem`,
+      Key: `${userId}/private.pub`,
```

Comment:
Este sería `public`, ¿no?
```suggestion
      Key: `${userId}/public.pub`,
```